pages/vpub_0xf/vpub_0xf.metadata

```diff
@@ -0,0 +1,4 @@
+slides:
+  - input_file: "pages/vpub_xf/x-amd-opensil.md"
```

Comment:
@Stanislaw-bnk and here? You didn't test your changes after modification; otherwise, this would fail.